editor-std/ui_data_pt.txt

```diff
@@ -1,13 +1,13 @@
 #Menu
 User = Usuário
 Join... = Registrar...
-Login... = Logar...
+Login... = Conectar...
```

Comment:
Did Login should be Logar? Conectar is Connect

Reply:
"Logar" is more of an internet slang and not an official word in portuguese. "Conectar" can be used to express both "Login" and "to connect".

Reply:
> 
> 
> "Logar" is more of an internet slang and not an official word in portuguese. "Conectar" can be used to express both "Login" and "to connect".

Ok, got it